crates/symbolicator/src/services/symbolication.rs

```diff
@@ -1909,65 +1913,40 @@ impl SymbolicationActor {
 
     /// The actual stackwalking procedure.
     ///
-    /// This is a method, as opposed to a closure in
-    /// [`stackwalk_minidump_with_cfi`](SymbolicationActor::stackwalk_minidump_with_cfi),
-    /// to circumvent a problem with `rustfmt`.
+    /// It can be switched between using the old and new stackwalking methods
+    /// with the `use_new_method` parameter.
     fn procspawn_inner_stackwalk(
         cfi_caches: Json<CfiCacheModules>,
         minidump: Bytes,
-        spawn_time: SystemTime,
-        compare_stackwalking_methods: bool,
+        spawn_time: Option<SystemTime>,
+        use_new_method: bool,
```

Comment:
This is rather minor so I won't mind if you don't want this: but I'd be tempted to make this an enum instead of bool just to make the calling code easier to read.